src/xmipp/libraries/data/pdb.cpp

```diff
@@ -501,18 +501,29 @@ void PDBRichPhantom::read(const FileName &fnPDB, double pseudoatoms, double thre
 			// Extract atom type and position
 			// Typical line:
 			// ATOM    909  CA  ALA A 161      58.775  31.984 111.803  1.00 34.78
-			atom.name=line.substr(12,4);
-			atom.atomType = line[13];
+			// ATOM      2  CA AALA A   1      73.796  56.531  56.644  0.50 84.78           C
+			atom.record = line.substr(0,6);
+			
+			if (const char* errmsg1 = hy36decode(5, line.substr(6,5).c_str(), 5, &atom.serial);errmsg1) {
```

Comment:
this should probably be
`if (auto* errmsg1 = hy36decode(5, line.substr(6,5).c_str(), 5, &atom.serial)) {`
similarly below (line 515)

Reply:
ok

Reply:
done